### ISSUE TITLE
azul didn't create a window

### ISSUE THREAD

**Issue by IceCreamChocolateCandy** [fschutt/azul#361]
When I run the example on macOS, I get an output `azul.App.run(x11)` in the terminal, but there is no window. I'm using version `azul-d54e1d94090e89b78e3ddcefd6372dcf6b5e317f` and `1.0.0-alpha4.` How can I fix it?
<img width="1440" alt="2023-12-13 10 32 01" src="https://github.com/fschutt/azul/assets/118348693/60c405d4-a070-418b-ba36-80ab87b19c59">

**Comment by fschutt** [comment]
You can't, window creation is currently a stub on macOS. It's not implemented yet, only Windows works.

My current development plan is developing in Parallels (on a Mac, but using Windows). Once Windows works 100% I'll port it to macOS. I had to reimplement the window handling due to winit not implementing support for native menus (menu bars / context menus).

**Comment by IceCreamChocolateCandy** [comment]
Is there an old commit that can be build, run, create a window, and place HTML on it can running on three platforms? Can you tell me which commit it is?

**Comment by fschutt** [comment]
@IceCreamChocolateCandy no, azul is not mature enough for that yet however, I've done this here https://github.com/projekt-dgb/dgb/blob/b521304d71b55979ec67bf76808e88069503131d/src/main.rs#L6923-L7040 (uses the wry and webview crates), so you might copy from that
